packages/mdc-toolbar/README.md

```diff
@@ -136,6 +230,82 @@ The provided modifiers are:
 
 | Class                                | Description                             |
 | -------------------------------------| --------------------------------------- |
-| `mdc-toolbar--fixed`                 | Make toolbar fixed to top of screen.    |
+| `mdc-toolbar--fixed`                 | Makes toolbar fixed on top and have persistent elavation |
+| `mdc-toolbar--fixed-upgraded`        | Upgrade toolbar dynamically add waterfall effect |
```

Comment:
Not sure we want to document this here because it's not a class we want users manually attaching